### DISCUSSION TITLE
Accepting RGB files to encode instead of only monocromatic

### DISCUSSION THREAD

**Discussion by migueldinisdev** [migueldinisdev/prototype-compressor#4] [Ideas]
How can we achieve this? Currently, the software reads monocromatic images in uint8 which is quite restricted.

Thoughts on this, @EdgarTip @IronMan988

**Comment by EdgarTip** [comment]
We would need to adapt a lot of our code and create new functions to achieve this goal. We would have to firstly check if it is a non monochromatic image and then direct the main code to a special encode function.

**Reply by migueldinisdev** [reply]
Yes, absolutely. Too much work, indeed.
